mne/io/boxy/boxy.py

```diff
@@ -295,7 +299,7 @@ def __init__(self, fname, datatype='AC', preload=False, verbose=None):
                                           rpa=fiducial_coords[2])
 
         # create info structure
-        ch_types = (['fnirs_raw' if i_chan < np.sum(mtg_chan_num) else 'stim'
+        ch_types = (['fnirs_ph' if i_chan < np.sum(mtg_chan_num) else 'stim'
```

Comment:
here is where it needs to be conditoinal on data type imported

Reply:
done, if we ask for phase data it will change the type to fnirs_ph, else it will stay as fnirs_raw

Reply:
great